src/wasm.h

```diff
@@ -86,6 +86,10 @@ struct Name : public cashew::IString {
   static Name fromInt(size_t i) {
     return cashew::IString(std::to_string(i).c_str(), false);
   }
+
+  bool hasSubstring(cashew::IString substring) {
```

Comment:
I'm all for making wasm.h more minimal.

I'd like to go much farther, and move pretty much everything out of `wasm.h` into separate header/implementation files. So move `Module` to `wasm/module.h`, `Const` to `wasm/const.h`, `Literal` to `wasm/literal.h`, etc.
Then because I'm sure it'd be convenient for some files to know about all the things currently in `wasm.h`, leave `wasm.h` where it is, but replace its contents with `#include`s to everything that gets moved out.
Does that sound like a good idea at all? If so I should do it in its own PR I think.